src/inbox/classes/IterableInboxDataModel.ts

```diff
@@ -258,13 +258,13 @@ export class IterableInboxDataModel {
   ): IterableInAppMessage[] {
     let sortedFilteredMessages = messages.slice();
 
-    // TODO: Figure out if this is purposeful
+    // MOB-10424: Figure out if this is purposeful
```

Comment:
Isnt this in reference to '==' which is not used here

Reply:
`!=` is used here -- it needs the same review